vstc/src/test_inputs.rs

```diff
@@ -32,11 +32,11 @@ mod tests {
       let file_contents = fs::read_to_string(&file_path).expect("Failed to read file contents");
 
       if let Some(first_line) = file_contents.lines().next() {
-        if first_line.starts_with("// test_output! ") {
+        if first_line.starts_with("//! test_output ") {
```

Comment:
This won't match the new format, the start of the pattern is `"//! test_output("`.

Reply:
Optional: I'd also suggest that we filter on `//!`, and then if the pattern isn't recognized, we fail the file for that too.